packages/ui/v2/core/Shell.tsx

```diff
@@ -282,7 +282,7 @@ function UserDropdown({ small }: { small?: boolean }) {
                   </span>
                 </span>
                 <Icon.FiMoreVertical
-                  className="h-4 w-4 flex-shrink-0 text-gray-400 group-hover:text-gray-500 rtl:mr-2"
+                  className="h-4 w-4 flex-shrink-0 text-gray-400 group-hover:text-gray-500 ltr:mr-2 rtl:ml-2 rtl:mr-4"
```

Comment:
Is it correct that on ltr it only has a margin on right side, and on rtl it has a margin on both sides?